_templates/how-to.md

```diff
@@ -0,0 +1,47 @@
+---
+title: [Title]
+description: [Brief description of the How-To article]
+type: [HowTo]
+<!-- TO DO: Do we we need the below fields for How-To articles? -->
+[teampulseid: [number]
+[ticketid: [number]
+[pitsid: [number] <!-- Not sure we need this, as pits item are related to teampulse anyway -->
+<!-- TO DO: What else should we have here? -->
+---
+
+# Title
```

Comment:
This is not actually mandatory. We can have this one automatically populated from the title meta field. It is worth discussing whether we want to pull it automatically, or if we want to have some degree of freedom and be able to tweak the title in the body.